src/components/AccountForm/AccountDetails/AccountDetails.css

```diff
@@ -0,0 +1,26 @@
+.account-details-container {
+    padding: 40px;
+    width: 80vw;
+    max-width: 400px;
+}
+
+.account-details-user-header {
+    display: flex;
+    flex-direction: column;
+    margin-top: 20px;
+}
+
+
+.account-details-actions {
+    display: flex;
+    flex-direction: column;
+}
+
+label {
+    margin: 4px 0px;
+}
+
+input {
+    margin: 4px 0px 20px;
+}
+
```

Comment:
nice
This file could be using 2 spaces. I think some of my older .css files were never converted to 2 spaces but it should be 2.